app/controllers/users_controller.rb

```diff
@@ -18,4 +18,9 @@ def follow
     end
     render :follow
   end
+
+  def user_delete
+    user = User.find(params[:user_id]).destroy
```

Comment:
admin以外もリクエストできるので非常にマズいです。viewでやってるのはボタンの表示制限だけなので、コントローラー側でも権限制御してください。

```suggestion
    raise "管理者限定機能です" unless current_user.admin
    user = User.find(params[:user_id]).destroy
```